package.json

```diff
@@ -65,6 +69,7 @@
     "multicodec": "~0.5.1",
     "multihashes": "~0.4.14",
     "ndjson": "github:hugomrdias/ndjson#feat/readable-stream3",
```

Comment:
We are shipping a non versioned module to users? If ndjson is not interested in the contribution, please create a fork and maintain it.

Reply:
Yes this needs to get resolved. I think it's not so much the at PR is not welcome it's more time commitment. Max added me to `concat-stream`, perhaps I can also get admin here as well ;)